ui/handle_input.go

```diff
@@ -167,10 +168,27 @@ func (win *Window) HandleInputs() {
 	}
 
 	if win.inputs.toTransactions.Clicked(win.gtx) {
+		win.wallet.GetTransactionsByWallet(win.walletInfo.Wallets[win.tabs.Selected].ID, 0, 100, 0, 0)
 		win.current = win.TransactionsPage
 		return
 	}
 
+	if win.combined.transactionSort.Changed() {
+		win.wallet.GetTransactionsByWallet(
+			win.walletInfo.Wallets[win.tabs.Selected].ID, 0, 100,
+			int32(win.combined.transactionStatus.Selected()),
+			win.combined.transactionSort.Selected(),
+		)
+	}
```

Comment:
Changing shouldn't the window to request the same transactions in a different order